tests/drivers/spi/spi_loopback/testcase.yaml

```
@@ -182,3 +182,10 @@ tests:
     extra_args: DTC_OVERLAY_FILE="boards/nrf54h20dk_nrf54h20_cpuapp_fast.overlay"
     platform_allow:
       - nrf54h20dk/nrf54h20/cpuapp
+  drivers.spi.kinetis_flexio_spi.loopback:
```

```suggestion
  drivers.spi.ke1xz_flexio_spi.loopback:
```
because the overlay is `frdm_ke1xz_flexio_spi.overlay`

---

updated